rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/InsertEventCountQuery.java

```diff
@@ -31,7 +31,11 @@
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.EVENT_COUNT_COL;
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.ID_COL;
 
-
+/**
+ * Query that inserts a new event count to the table with event counts.
```

Comment:
Also, pls add a good doc to Aggregate storage (and other storages) about how many tables we have (one for aggregate records and another... etc).